lib/evmone/instructions_calls.cpp

```diff
@@ -180,6 +180,66 @@ evmc_status_code create_impl(StackTop stack, ExecutionState& state) noexcept
     return EVMC_SUCCESS;
 }
 
+evmc_status_code create3(StackTop stack, ExecutionState& state, code_iterator& pos) noexcept
```

Comment:
Currently there is a lot of overlap with `create_impl`. Would it make sense templatizing it with some `if constexpr` branches?